syntaxes/vba.tmGrammar.yml

```diff
@@ -34,9 +35,9 @@ repository:
       - name: keyword.conditional.end.vba
         match: (?i:\b(End( )?If|End (Select|With)|Next|Wend|Loop(( While)|( Until))?|Exit (For|Do|While))\b)
       - name: keyword.control.vba
-        match: (?i:\b((?<= )E(nd|xit)|As|And|By(Ref|Val)|Goto|Is|Like|Mod|Not|On Error|Optional|Or|Resume Next|Stop|Xor)\b)
+        match: (?i:(\b(Exit (Function|Property|Sub)|As|And|By(Ref|Val)|Goto|Is|Like|Mod|Not|On Error|Optional|Or|Resume Next|Stop|Xor|Eqv|Imp|TypeOf|AddressOf)\b)|(\b(End)\b(?=\n)))
```

Comment:
Good catch with the tab issue.

Reply:
@serkonda7 Where should I add the links? In this PR?

Reply:
@stepanhakobyan yes, just post a comment here in this pull request (e.g. reply to this discussion).